modules/runner/src/integrationTest/java/org/apache/ignite/internal/configuration/storage/ItRebalanceDistributedTest.java

```diff
@@ -391,6 +435,85 @@ void testRebalanceRetryWhenCatchupFailed() throws Exception {
         assertEquals(3, getPartitionClusterNodes(2, 0).size());
     }
 
+    @Test
+    @Tag(TEST_TX_STORAGE_TAG)
```

Comment:
>Tags are used to filter which tests are executed for a given test plan

But here, they are used to signal that some customizations need to be applied in the `before()` method. This looks like an abuse. It seems that a custom annotation should be added instead.

Reply:
Fix it